draft-ietf-tls-tls13.md

```diff
@@ -1395,7 +1395,7 @@ Structure of this message:
        uint8 CipherSuite[2];    /* Cryptographic suite selector */
 
        struct {
-           ProtocolVersion max_supported_version = { 3, 4 };    /* TLS v1.3 */
+           ProtocolVersion legacy_version = { 3, 3 };    /* TLS v1.2 */
```

Comment:
Why do we persist in using a struct for the version, rather than just changing to a uint16-based enum?